string rover_mode
sensor_msgs/NavSatFix[] GPS_goals
bool GPS_GOAL_REACHED
bool MISSION_OVER
bool AR_TAG_DETECTED
uint16 curr_AR_ID
bool MANUAL_ENABLED